src/main/java/ca/corefacility/bioinformatics/irida/ria/web/launchPipeline/dtos/LaunchRequest.java

```diff
@@ -14,6 +17,45 @@
 	 */
 	private String description;
 
+	/**
+	 * List of file identifiers to run on the pipeline
+	 */
+	private List<Long> fileIds;
+
+	/**
+	 * Email the user when the pipeline is completed
+	 */
+	private boolean emailPipelineResult;
+
+	/**
+	 * Write results back to the project the samples came from
+	 */
+	private List<Long> projects;
+
+	/**
+	 * Update the samples run on the pipeline with the pipeline results
+	 * Ignore if not required.
+	 */
+	private boolean updateSamples;
+
+	/**
+	 * Identifier for a reference file to use on the pipeline.
+	 * Ignore if not required
+	 */
+	private Long reference;
+
+	/**
+	 * List of parameters for the pipeline.
+	 * Should be: key: name, value: value
+	 * The value needs to be an object because it can be boolean, string, number, etc...
+	 */
+	private Map<String, String> parameters;
```

Comment:
Change this Map<String, String> to be a new dto called PipelineParameters or something

Reply:
It needs to be a `Map<String, String>` to be used by the `AnalysisSubmissionService`.  this was an intentional choice.

Reply:
Ah ok. All good then